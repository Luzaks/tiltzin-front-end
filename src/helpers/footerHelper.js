import React from 'react';
import { Icon } from '../styles/StyledComponents';

const footerHelper = icon => (
  <Icon key={icon} className="social-icons" src={icon} alt="icon" />
);
export default footerHelper;
