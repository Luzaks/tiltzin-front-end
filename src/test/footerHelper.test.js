import Twitter from '../assets/imgs/twitter.png';
import footerHelper from '../helpers/footerHelper';
import '@testing-library/jest-dom';

test('the type of the returned footer helper.', () => {
  const Icon = footerHelper(Twitter);
  expect(typeof Icon).toBe('object');
});

test('the rendering footer helper Icons .', () => {
  const Icon = footerHelper(Twitter);
  expect(Icon).toMatchSnapshot(`
     <img src={'../assets/imgs/twitter.png'} alt="icon" />
  `);
});
