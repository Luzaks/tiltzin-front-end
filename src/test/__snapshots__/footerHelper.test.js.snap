// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the rendering footer helper Icons .: 
     <img src={'../assets/imgs/twitter.png'} alt="icon" />
   1`] = `
<ForwardRef(styled.img)
  alt="icon"
  className="social-icons"
  src="twitter.png"
/>
`;
